playbook/app/pb_kits/playbook/pb_icon_circle/_icon_circle.scss

```diff
@@ -57,13 +57,11 @@ $pb_icon_circle_sizes: (
       border-radius: $size/2;
       background: $silver;
       color: $text_lt_light;
-      font-size: $size * 0.38;
+      font-size: if($name == "xxs", $size - 6px, $size * 0.38);
       line-height: $size;
       flex-shrink: 0;
       flex-grow: 0;
       flex-basis: $size;
-
-      @include svg_size($name, "xxs", $size - 6px);
```

Comment:
@ASPhillips8 I did a quick search. It looks like this mixin is no longer used anywhere else. This was the only location it was called. So you can cleanup the code by deleting the mixin above for `svg_size` as well. I would just make sure to test it well again, in case there was some other spot relying on it.